dev/com.ibm.websphere.appserver.features/visibility/public/appSecurity-2.0/com.ibm.websphere.appserver.appSecurity-2.0.feature

```diff
@@ -16,3 +16,4 @@ Subsystem-Name: Application Security 2.0
 kind=ga
 edition=core
 WLP-InstantOn-Enabled: true
+WLP-Platform: javaee-7.0,javaee-8.0
```

Comment:
can you double check these values?  I see appSecurity-3.0 also uses javaee-8.0

Reply:
Removed javaee-8.0